.travis.yml

```diff
@@ -1,13 +1,17 @@
+dist: xenial
+
 language: php
 
 php:
-- 7.3
+  - 7.1
+  - 7.2
```

Comment:
This was actually intentional. The tests in this project are run inside docker containers. This is done because this project is cross-platform and we don't have the capacity to implement multiple testing frameworks in multiple languages.

The PHP part in this CI is only to trigger the tests that are executed inside the containers. Using only the latests PHP version is enough and by running the tests only on one env we save a lot of travis CPU time.

Reply:
@eldadfux, thanks for your reply and explanation.

Revert this change and using only latest stable `PHP-7.3` version :).